### DISCUSSION TITLE
Welcome to Round-Anything Discussions!

### DISCUSSION THREAD

**Discussion by Irev-Dev** [Irev-Dev/Round-Anything#18] [General]
## 👋 Hey!
Use this space for anything that's not a bug.
  * Ask questions you’re wondering about.
  * Share ideas (for you own project or a feature.
  * Share something you made with the library.

Or just say hi.

**Comment by wadebee** [comment]
Hi I'd like to use your library but I am new to OpenScad and my use-case is very basic. Your examples are all too complicated for my needs and my lack of skills at this point. 

Would it be possible for you to include some simpler examples? My use-case is just to build an organizer box (for 3D printing) that has rounded edges along the sides and the bottom 4 corners fully rounded (sides and bottom).

**Reply by Irev-Dev** [reply]
That's not a bad idea @wadebee, I don't really have time to add those anytime soon though.

I guess the intention with the examples was to demonstrate things that are normally very difficult to do in OpenSCAD without this library. Rounding the edges of boxes or things with 90 angles is not too difficult in vanilla openSCAD, you could try hulling 8 spheres to a rounded cube.

**Comment by baryah** [comment]
Wonderful library. I was wondering how to add fillets and deepen the inter tooth space a bit in the spur-gears by http://www.thingiverse.com/thing:3575.  Round-Anything came to the rescue and I could modify the script to do what I wanted.  The modified script is loaded at - 
https://www.thingiverse.com/thing:4917647.

**Reply by Irev-Dev** [reply]
Hey @baryah,
That's great, what do you plan on using the gear with?

If you felt like doing me a favour you could sign up and add the gear to [CadHub](https://cadhub.xyz/) too? I'm currently trying to get the website populated with a few examples so you gear would be most welcome. We don't have customizer support yet (that's comming) but the IDE support is pretty good at this point. One quick note is that the include statement needs to be `include <Round-Anything-1.0.4/polyround.scad>` on CadHub

[Here's an example](https://cadhub.xyz/dev-ide/openScad#encoded_script_v2=eJzlG9lu20jy2fqKnmB3QsmkKMlxDmUYw2M7BzYXbM/Mg2EIFNmSuOGhYVO2NYE/aH9jv2yr+mI3RStOMoMFdoXEELu7qqvrrmrK998VcTJLaExmZZGRakGJ1+n4PvkYlmFGqzKJyJv8qkhXFSU/0yuakjCPydlyVZJXNCwZma7Jq5LOX5YFqxDwTUUSRtIkojkDtKs8piXHe1TSsEqu4EuRZUXOiEdevf+FvPr4Vq3uI/y//0VGg+FgA++iqpZj37++vu5XiySfA6YSQKIi8/nzeG//yT4sFAeKYKsiRyTHJXkdlp/CvCJnsGzR6SR5lK5iSn46LYA67zBfcwTesD/oP/KXRboucaLPojB+AczokYuPZZKF5Zo4J4zRvErCtHtJej5s974gxYxUlFYLkuT8oHPgS7+Dfyf5KpvSclLMJmJFQPYGzwHqHJbFCTIYmIPwEkxIoVjhMI3nFOeKnJKqKAC6KvhCexrQzIs8TNM1KZbLgiWVXC5J4KsnerOAPOUUfCwpY6uSgjjnKTVpkHBLuWAiFgRktA9wvYsPsK5kl/zw54sk+pTDuhb4Ss8FZMi3PG4c+PVq2gK3WE1NaiW7GvsgqHO2KFZpTKaUZAUcpFqEgv8P7f0fuoSmjJKkItcFaAGsv0pYMk2psaNJbSuxoKBVGaYkSspolYKkprCngMdvGyTnXC/06mhVeSAJpnRkWsRrXMCF7gxJXFBG8gJJLD91+x0ETCmbABxKcJLkEw4isR9ymTgxnZeUsi4JARBOsKgVSegb0yxCpU9yMMiq6Heq64RVCtdpGCcrzdlZkqa0Ig6YHhhQF9BSzlsLbQZ/La5w2jxCb5YULAUGyXVSCWqECaDugv0DhWEU0WUVAvtJTFkyz6UUxMaTUlADOtPf53KgMyCbcVQsQ7029UCQ4xFg8Vy6mbxw5UI5wNf0weqFF5vMwohWWs5/m+UADy5jBucEuq/CdAXbLcsiXkXwpZ5JaT6vFhtbX4FfAO+BrmZWhhG6Hb4mJFnWJ6wA5leoFyhcYMIKNBH2dLkbLUNOJY41yYOlIDJgGUM+yS9gkrgIPRuCL2geocRy2rRBpk7d6QgN6+w0XFHQ5p/czo7S2MkyqaJFEIUpPgJd9oTTCk5aPE4XkNq+JGjxL7DIttrAfoR5UCxjejiQQ9dJDKfhj5YhB5u2LZcoyoINfwMLLGMONu1bsghsM7jDRhFJGH1KQ7YIkCpubAH/C08NOQeNZ7dD4GPZQrBpHt3nnc4yCfb6w0fD/Wejx/t7+0+fPXm2N9p7+ujx6PGjvb2ne6Mnz8CAOhnocSqNt0ULhvubMp+F4C5dGZ/Auq3hDWGOniKGFPCHqI1gVv3RpjT3NwT41JLf/ob4tES1MDittnz2DXEM2hk/aGH6oI3Nw27nMyydEafBD3Fy8uOPGywRM13S2dmh0aJwHrw7OjwmJ6enH07HguWS/TmlMSM04eYeNvGQoiT2ng9AwkihwKqjiNj1gWsvRm3Y8f2jIoeUqFLIIUcR69GVFJCu5SL7UrCemPTINMQ8DVyKjCau9CvDpwNfEypWu+CqVjzVsEnCjG+Z1KvFaL+pWaAcDd7+IDh4YI+Ora2tQ5KPHJFSgDEx47TYROgD7JUbqZlPGsRxB4pj9ekC3LbJHo9R2hAOgZyTx7hwWlzRPjkrBEovbtLiCVr0RJAXXjHzOEk9eycfNgeG8e91MgGnIM0cstckx0dRPVewOoDaqPwRrEB1Is6Dc8QzfuA2UbvkgeSvQIJLTKRKBj9jLDriR0OTY7S5qXjsRQVzbG+hMBw3dHRM3os0QQtsiXExT1TYbbCGQwYbrPEbh5a7HcYxzeNVNiaY6wBkDI6Beyuebltqw42lYklM5QjsHEp4zEr8Jhlijw88J1c8ERGwlSm7Cplp4CKjl37ogWuCS0cguEa/6hwlJEHaUjo7MdWn0AfaJdpzS16dmkC2eU2LqsLqZCbLEbYEb8qAP7hR+2k9tak4BZRLH87J2S+nJ+T1h9/Iz4dH/3h7ePaavDkj704OcfyYHL4/3pg8+vD+15PTc5g9/1BPHL5/9fak9vm6UFEDWh0kaT20Ez6oqCEz8Lx/0LLQMOD8wnQmwpBECL4NS7AZ+FA4tEyyxFRIWMK/NNxhFn6idQbMk2TwExDOmrjBIe49RpqamuwR+1hdWPNIiujXEPQO8mcGjggwF2mK+SfuBPG0L6KqloYpG4/ogPtFVPpEiwKnrdJWhtxJUU6Tqg7GZrD2R7s1Gc8bENEKcGc8rY5osEx6rfgkiW+TLKk0ScAzlvzB1fvZ4O926Qza2D/47fXhOVeZN6dHb08mx28O352cn5zWRGtyIWhncCxIinZ2Bv0ng97dNPoyx3BxrVMfzLOP3O0N+s/QxamiIaIOCM6vQB0YuhWjwmJYs4lKsN2qIBep0RCn29mBkR1wiRCp8Yk/NpfI0R0s9SA+0JuqxE6Hs6DJfFEFVgbmosyv6E1SrTHZIkaeyjGJFI4twiUVTNppZo+uGG2NO+JRrrAdhPEk5+0YYjzJ+YY/NR/lihbLag7JlV9IwHmEwmWQBzb6GD8Ri3+CSTuQP+QMKyTiXAzcgWsDXcpV0RpEgp0wpwxq/XEXtkg8G3h3CNQA8G2nnaAXbQTduZMNLDAjUrv98YI01nXufUhzZ9swYHdrF6+FFjxkcxtvyFHXiHGfMrBqAH/EBbsIRrtZeOPYKm6Xf41d+bYq3wfrfjHQZgVxwUmCi8FYTiEhOx2hyhXQJ8hLeuC8lW+4bHDKuWj1akCuPld9MuBYwz1xpuGBGnWwLfHdPcW7W5EmyLiG/YKKWu4GQx1vwKj+Re1tZJMH3Cw2dkSvhScPMgMi7wtA5lkOioQAQX+HakC1JY10uy/KK9jRmYMlXgzHDc8Bxyfy81l/M1k756y9G+q7HZxCpM3fqLJBz8iWz1Z/tw1wixvcBrbFO24D2+o0twHey5duQ3CvHsddH7sot562AzZbG41nmUjj57YDn1v+D+864jK8ZmaGgZmbqoFIyCy5YdkYQQnMGpojbEOW+2ARiBTyQzEMod40vbQQ1xOMOMBNzLK6aso0ImI2b+pAvBmEGxHXjq92NG2EzpYY2YyHmIjwrEOlIMJMweQasF3FPudvs7zJ/d7IJRCPasoMcfh+AkQhTuB1FuZrUiUZ5Zzf8CvcNRMnucuroP82lcL3naTNlXTBneEVEjOyeZBayB0LmRciDS6L1dzM4UVfV8oUFGEZMmaurFfprFQXkTXjjNjRdgCZwrWpl/O/mXDJFIdHsdvaIuuLL5kqK7sB2cn6S117JPzSEcEgOeZ3QPdh9kPeyUFByvJG2ptnWFxfGeEd8vjLbQ9qlFoQkBAQx8RqW5QM/4KmJShxRQ4/fjw5PD1D5Z6KhAD/i7lCnHpptHyM+x5FTH0zjBvXPjJNxT0Q8N4jN+5at2gE8qBGYFFcMxJW4RVuVEkNspdZjafu8/poi7ouN2wXv2E7Th4t4yUXL8Fl+X1woOhT+hpWYT4ijkH0xfBSbSyeB5d6Z2nbem8Un90N4My9ls1AQQcqMXgMcXW2ECGipTfgErbil3ehfa0m8NYiEVqXMClNoe+73BV5EpU83TW/90MPp8nCC2O2gYxVYVmJsAaCpHmMDR0LlXIw4ml3w8wVgzgmucozmyTm3aS5aWfHBAnuqxi1QXTrrYul3lkms4wfRrkO0WHTztgAuPe+LV053y8RgXEtZyXXHonpklOxWkpbE8FM3vFx+WHDhrLAadj+D8Gge9AYG/PIihP7Y/jm84ZQ3RH4rKNjALGx5CWJioPNqKjv9bR6cM0VOutERVHGSQ7TDC27K1zAFtXRBQQ/rcddN++herX3sAxI4mrRZ4wGnI5h0O5ADKXZdSzJGzPdnpN4wy7IRwYYjnL0fSgTic5k5eeNnBTUEU4+DGQE0u6WvypjWZa+E0d2qfc5NH959OSoJpIfAqP0r46JyhVbyH6BCTX6MtSocSSZk4nbpNqxGgqCmVJZonNR5qXVRZ5Av4pC5/EdxxqpY2VJWRalIvAehzROOVKn/Eoko27LoTcGPm8OieTEMnKu4VK7ZS0s+RRTFpXJlMaShR7Ie0mRZxg9ceE1tldtr5HUjoHrbbqeF7lO/Tji4ALTmkvX1A7zYeSavDEfIMBJRGG1EHiG7sjdcx8BvssWntx22p/EN5mv6StEI4ky86Vqoh2izuy4e7BfOoG0juhej+cR7aTk6x8U6vlYBnjlQIgWh3jjwzO+gzNSO0wLoCBr632AUXJtke9weC2ZorgrVPtIMjShOPlPoEkUAmFGVrwmWKzS1Ol+vr0jgdSdhr80g2y55xZDzXKoU/sySbhlEHj4SdsBJlyMdr/k7obInS2PO5sad3cttrcktrYctjQVcnojziSv9p/XbTWromhTEyPc/h/yqypXwC7bYd4+l15CvMPHlG8Io0q8RrDRevGk9XBDCzPwpGWSR8kSgUTnhXsE3jyR/YG7/U2f/MLwXS5wofxVT6hXIAmP0czflPTqmF4RB18rZWPfn0NGvpry10lxzoNJ334/tGt5G9n17ZOHmgUPxctjUIACK3z5akiMzdyMv8ImsmDlgiCaIpCOmBKhmQr1yUv+5klGw5zVuxfyrm++qGyHhhxyCe7Om7WiZqYz8DKa4arNBDQLoK72T1/U2f+Sp1LsVT7qqyriv7Qk/cZ68lvKqz+jVvozqp7vLFbuqEeSYGhnHhDyK7Nb+JNBk71SjAn2Bgh2YICNLbC2wsIE/7Mqi2/B2V5afF8F8N0JdiP1sx5xA8l1baEH5q5jk/CGzOy2vj15s24gFCh2IeyW/uhyXA95ckjC2+RBpKDOxRCy230wus08XubVjg4OzgWkdVFYOTfr3QsP79/wz6V7UV52XWNq+0zrhGfNXAJFvPrYyKXFdQR5h28FZxDk4Azk5SoX7xTDTGB/+OKXCVbcdbdFOnitjVB4yPcOePdO3WKIIewP5fVbRFJzZbmfVA8ZL2toiT/MYMWqjOjY/B0G0LhY0HQ5K8pVxkMnjng45M9pgS9Irf0hZErDoXoTTtHlsSK9wh9drL0wX3ucEO+mv6iytNOZyTPf10cpsgPCfoeSFWR6rc7iWwtH2PYfdq03gZCJR1Y3RPOOl7BIB3ZVQskti428daw4jZT1DeqFocUTwzPwEZe0Bx95ixKQzgXEiIKp9UjvlwKXgIX4AhGE4VstXw3KK8Nv23V4ie7sm3YdXHbAfmue2a6K96FqjsADALgQDD3xMGxA265NcVthaWGrwrnJNbXBJlPUzOaZFTabqPtxQh7SWNJz+MbNdbsNyN4ywXdHe5yWxuKu28DYtgjO0Y6xbXvAaB2u/n0ATxhlKvkVvw4AsaDza5nyndFuGx5Vk7XQ8M2/UZBUfPdpevgm7n8An+1Ecg==)
